modules/core/02-client/legacy/v100/store.go

```diff
@@ -139,3 +148,36 @@ func pruneSolomachineConsensusStates(clientStore sdk.KVStore) {
 		clientStore.Delete(host.ConsensusStateKey(height))
 	}
 }
+
+// addConsensusMetadata adds the iteration key and processed height for all unexpired tendermint consensus states
+// These keys were not included in the previous release of the IBC module.
+func addConsensusMetadata(ctx sdk.Context, clientStore sdk.KVStore, cdc codec.BinaryCodec, clientState *ibctmtypes.ClientState) error {
```

Comment:
I decided not to modify `PruneAllExpiredConsensusStates` as that seems like a useful ibctmtypes function by itself. Since the insertion of the iteration key and processed height key is logic specific to this migration, I decided to iterate the consensus state again and do it in a separate iteration. It is less efficient, but I think better code wise. Open to changing if efficiency is a big concern